### ISSUE TITLE
Will you release the instruction data generation code or the instance generation code？

### ISSUE THREAD

**Issue by SVT-Yang** [jihaonew/MM-Instruct#3]
Thanks for this wonderful work!  Will you release the instruction data generation code or the instance generation code？

**Comment by jihaonew** [comment]
Yes, we are working on releasing the data generation code. Please stay tuned.

**Comment by DWCTOD** [comment]
> Yes, we are working on releasing the data generation code. Please stay tuned.

大佬，大概什么时候更新哈，很期待

**Comment by WhitEiller** [comment]
When will the instruction data generation code be released? Thanks
